challenge_take_out_the_garbage/package.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
```

Comment:
Same here, just remove all the outcommented stuff and please fill in the templated fields